### ISSUE TITLE
Is the learning rate negative?

### ISSUE THREAD

**Issue by AlephZr** [WANG-Chaoyue/EvolutionaryGAN-pytorch#8]
If my understanding is correct, after 200 epochs, the learning rate of the network will drop to a negative number

**Comment by AlephZr** [comment]
It's my fault, update_learning_rate was not called and the learning rate did not change
